app/src/test/java/com/concordia/mcga/activities/PathFinderUnitTest.java

```diff
@@ -0,0 +1,34 @@
+package com.concordia.mcga.activities;
+
+import com.concordia.mcga.utilities.pathfinding.PathFinder;
+import com.concordia.mcga.utilities.pathfinding.PathFinderTile;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class PathFinderUnitTest {
+    @Test
+    public void testShortestPath() throws Exception {
+        PathFinder finder = new PathFinder();
+
+        List<PathFinderTile> tiles = finder.shortestPath(1,1,4,4);
```

Comment:
We should also test this code with at least two maps, one that has a solution, and the other that the destination is unreachable...